Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -137,41 +138,37 @@ WarpX::AddSpaceChargeFieldLabFrame ()
                                      "Error: RZ electrostatic only implemented for a single mode");
 #endif
 
-    // Allocate fields for charge
-    const int num_levels = max_level + 1;
-    Vector<std::unique_ptr<MultiFab> > rho(num_levels);
-    // Use number of guard cells used for local deposition of rho
-    const amrex::IntVect ng = guard_cells.ng_depos_rho;
+    // Allocate fields for the potential
+    // Also, zero out the phi data
     for (int lev = 0; lev <= max_level; lev++) {
         BoxArray nba = boxArray(lev);
         nba.surroundingNodes();
-        rho[lev] = std::make_unique<MultiFab>(nba, dmap[lev], 1, ng);
-        rho[lev]->setVal(0.);
+        phi_fp[lev]->setVal(0.);
```

Comment:
This line was removed in a PR a while back to speed up the iterative solver - basically using the solution from the previous step as a starting point so that fewer iterations are needed to converge.
```suggestion
```

Reply:
Thanks for catching that! I will update it.